lib/popup.ts

```diff
@@ -172,6 +208,10 @@ export class PopupControl<T extends IPopupOptions = IPopupOptions> extends Dispo
     this._showDelay = options.showDelay || 0;
     this._hideDelay = options.hideDelay || 0;
 
+    // Set default for the `boundaries` option to `viewport` to match the description of
+    // IPopupOptions.attach.
+    options = defaultsDeep(options, { boundaries: 'viewport' });
```

Comment:
I am uneasy changing underlying defaults without clear understanding of what's different. If they do need to be changed, I think the interpretation of defaults would be better done at the place where `options.boundaries` is used rather than here (i.e. in `OpenPopupHelper` constructor, where it's already deciding how to interpret `options.boundaries`)

Reply:
Hmm, it was not my intention to override Popper's default options, only the documentation was confusing and tricked me into thinking that this line was missing. Instead, I'd prefer to override that default at the `popupOpen` level, where we know it makes sense, and to remove the misleading bit from the documentation.